bare-metal/elastic-metal/how-to/send-metrics-logs-to-cockpit.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: How to send metrics & logs from bare metal servers to Cockpit
+  description: This page explains how to send metrics & logs from bare metal servers to Cockpit
+content:
+  h1: How to send metrics & logs from bare metal servers to Cockpit
+  paragraph: This page explains how to send metrics & logs from bare metal servers to Cockpit
+tags: metrics grafana cockpit elastic-metal
+dates:
+  validation: 2024-02-13
+  posted: 2024-02-13
+categories:
+  - bare-metal
+---
+
+Monitoring your dedicated servers with [Scaleway Cockpit](/observability/cockpit/quickstart/) offers detailed information about performance and health, ensuring optimal functionality and proactive issue resolution.
+Cockpit provides a user-friendly interface for real-time monitoring of CPU, memory, disk usage, and network activity, enhancing the reliability and performance of bare metal servers.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have an Elastic Metal or Dedibox server running on a Linux Operating system
+</Message>
+
+## Activating and Accessing Cockpit
+
+1. Activate Cockpit on your Scaleway account if you have not already. Follow the instructions in the [official documentation](/observability/cockpit/how-to/activate-cockpit/) for further information how to do so.
+2. Generate a push token to push metrics/logs to Cockpit by following the instructions provided [here](/observability/cockpit/how-to/create-token/).
+3. Configure Grafana Agent to push metrics/logs to Cockpit. Refer to the [official documentation](/observability/cockpit/api-cli/configuring-grafana-agent/) for detailed instructions. 
+    <Message type="tip">
+        Ensure to set the interval to 1 minute for real-time monitoring.
+    </Message>
+4. Retrieve Grafana credentials to access your Cockpit dashboards. Instructions can be found [here](/observability/cockpit/how-to/retrieve-grafana-credentials/).
+
+Once you have the Grafana credentials, access your Cockpit dashboards by navigating to the Cockpit dashboard URL provided in the [official documentation](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).
```

Comment:
Maybe we just add the link to the documentation on accessing Cockpit dashboards to step 4, as it repeats a bit in here, and delete this sentence? Something like:

`4. [Retrieve your Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/), then use them to [access your Cockpit dashboards](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/).`